moto/resourcegroupstaggingapi/models.py

```diff
@@ -383,7 +383,8 @@ def get_kms_tags(kms_key_id):
             or "rds" in resource_type_filters
             or "rds:snapshot" in resource_type_filters
         ):
-            for snapshot in self.rds_backend.snapshots.values():
+            snapshots = {**self.rds_backend.instance_snapshots, **self.rds_backend.cluster_snapshots}
```

Comment:
Can this be a utility method, `list_all_snapshots()`? Ideally the resourcegroupstaggingapi doesn't need to know the internals of RDS

Reply:
I just noticed that for cluster snapshots there is a different ARN:
`arn:aws:rds:<region>:<account>:cluster-snapshot:my-cluster-final-snapshot`
Fixed